config/owasp/suppressions.xml

```diff
@@ -234,5 +234,12 @@
    <packageUrl regex="true">^pkg:maven/org\.apache\.commons/commons\-compress@.*$</packageUrl>
    <cpe>cpe:/a:apache:commons_compress</cpe>
 </suppress>
+<suppress>
```

Comment:
this suppression can be removed as it has been fixed

Reply:
resolved: [d6a2962](https://github.com/hmcts/ethos-repl-docmosis-service/pull/2300/commits/d6a296203a48d8b8dc29eaaf89a33f31fd61aa92)